Expose `dt_storage_pool_log_gc_force_hardlink_rate` to users. (#7815)
pingcap/docs: tiflash/tiflash-configuration.md

    ## New in v5.4.0. This item enables or disables the elastic thread pool feature, which significantly improves CPU utilization in high concurrency scenarios of TiFlash. The default value is false. The elastic thread pool feature is experimental and not recommended for production environments.
    enable_elastic_threadpool = false

>     ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). 

Reviewer: ```suggestion
    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.
```

>     ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a page file (log type). 

Reviewer: ```suggestion
    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). 
```

>     ## New in v5.4. Max valid rate of deciding a page file(log type) can do hardlink. 
>     ## Not enable by default. It is recommended to change to 0.8, if the GC time is too long.

Reviewer: ```suggestion
    ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a page file(log type). 
    ## Disabled by default. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```

>     ## Disabled by default. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.

Reviewer: ```suggestion
    ## Defaulted to 2, which disables the hardlink function. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```

>     ## New in v5.4.0. This item specifies the maximum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.

Reviewer: ```suggestion
    ## New in v5.4.0. This item specifies the minimum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.
```

>     ## New in v5.4.0. This item specifies the minimum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent the hardlink operations. The greater the value, the more strict the condition to trigger a hardlink.

Reviewer 2: ```suggestion
    ## New in v5.4.0. This item specifies the minimum valid rate for triggering a hardlink by a PageFile (log type). The value range is [0.1,1.0]. The smaller the value, the more frequent to trigger a hardlink. The greater the value, the more strict condition to trigger a hardlink.
```

>     ## Defaulted to 2, which disables the hardlink function. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.

Reviewer 2: ```suggestion
    ## The default value is 2, which disables the hardlink function. If the GC time is too long, it is recommended that you change the value to 0.8 to enable the hardlink function.
```

Reviewer 2: The word "default" is rarely used as a verb in user documentation
